crates/subspace-farmer/README.md

```diff
@@ -75,4 +75,25 @@ Windows
 
 ## Architecture
 
-Check it out [here](./ARCHITECTURE.md).
```

Comment:
Is it okay that it is duplicated?

Reply:
I meant to delete existing file, thanks for noticing!